src/overlays/actors/ovl_Dm_Char04/z_dm_char04.c

```diff
@@ -119,7 +126,7 @@ s32 DmChar04_OverrideLimbDraw(PlayState* play2, s32 limbIndex, Gfx** dList, Vec3
     Vec3f sp1C;
     DmChar04* this = THIS;
 
-    if (limbIndex == 6) {
+    if (limbIndex == FAIRY_LIMB_6) {
         sp28 = ((Math_SinS(this->timer << 0xC) * 0.1f) + 1.0f) * 0.012f * (this->actor.scale.x * (1.0f / 0.008f));
```

Comment:
```suggestion
        sp28 = ((Math_SinS(this->timer * 0x1000) * 0.1f) + 1.0f) * 0.012f * (this->actor.scale.x * (1.0f / 0.008f));
```
works and seems better?